iocore/net/quic/QUICGlobals.cc

```diff
@@ -41,6 +41,7 @@ RecRawStatBlock *quic_rsb;
 
 int QUIC::ssl_quic_qc_index  = -1;
 int QUIC::ssl_quic_tls_index = -1;
+std::ofstream QUIC::keylog_file;
```

Comment:
I'd rather have it in `SSL` context.

Reply:
Do you mean `QUIC::ssl_keylog_file`, `SSL::keylog_file` or something else?

Reply:
Talked offline, put this in SSL ex data looks reasonable.

Reply:
I followed what we're doing with `proxy.config.quic.client.session_file` instead of introduce new ex data index. Because these are very similar.